manifests/params.pp

```diff
@@ -115,6 +118,7 @@
     }
     $terminus_package       = 'puppetdb-termini'
     $test_url               = '/pdb/meta/v1/version'
+    $vardir                 = '/opt/puppetlabs/server/data/puppetdb'
     $database_embedded_path = '/opt/puppetlabs/server/data/puppetdb/db/db'
```

Comment:
This path should be getting it's value from vardir, `$database_embedded_path = "${vardir}/...`